psydac/linalg/tests/test_stencil_vector.py

```diff
@@ -138,6 +139,25 @@ def test_stencil_vector_2d_serial_dot( n1, n2, p1, p2, P1=True, P2=False ):
     assert z2 == z_exact
 
 #===============================================================================
+@pytest.mark.parametrize( 'n1', [1,7] )
+@pytest.mark.parametrize( 'n2', [1,5] )
+@pytest.mark.parametrize( 'p1', [1,2] )
+@pytest.mark.parametrize( 'p2', [1,2] )
+
+def test_stencil_2d_array_to_stencil( n1, n2, p1, p2, P1=True, P2=False ):
+
+    V = StencilVectorSpace( [n1,n2], [p1,p2], [P1,P2] )
+    x = StencilVector( V )
+
+    for i1 in range(n1):
+        for i2 in range(n2):
+            x[i1,i2] = 10*i1 + i2
+
+    xa = x.toarray()
+    v  = array_to_stencil(xa, V)
+
+    assert np.allclose( xa , v.toarray() )
```

Comment:
@saidctb Thanks for adding this test. Can you please parametrize `P1` and `P2` as well?

Reply:
done

Reply:
Thanks